tools/ci-scripts/check-rust-runtimes

```diff
@@ -27,7 +27,9 @@ do
     cargo test --all-features
 
     echo -e "## ${C_YELLOW}Running 'cargo doc' on ${runtime_path}...${C_RESET}"
-    cargo doc --no-deps --document-private-items --all-features
+
+    RUSTDOCFLAGS="--cfg docsrs" cargo +"${RUST_NIGHTLY_VERSION}" doc --no-deps --document-private-items --all-features
```

Comment:
This will overwrite the `RUSTDOCFLAGS` set in the `Dockerfile` that disallows warnings. Should we just add this flag there?

Do we also need to verify the docs build on stable?

Reply:
probably not...I think only docs.rs really matters honestly.

alternatively we could eventually add a separate job that checks the full SDK docs